README.md

```diff
@@ -1,9 +1,71 @@
-# bc-parks-adventure-mobile
+# Parks Adventure Mobile
 
-A mobile app to help users find and explore BC Parks.
+Find and explore the wonders of **BC Parks**.
+
+## Features
+
+Two of the main screens, `Explore` and `Find`, provide different ways to
+discover parks. Once a user finds a park they can find more information about
+that park by navigating to the `Details` page. Then if that park is of interest
+the `Favourites` page will help keep it close at hand. Users can also stay
+up-to-date on the access status of parks (open, close, etc.) and any alert or
+advisories they should be aware of before traveling.
+
+---
+
+### 1. Explore
+
+<div style="display:flex;align-items:center;">
```

Comment:
**suggestion (non-blocking)**: Strip out the HTML and styling. 

These look really good in your screenshot, but I don't think Github will render these for security reasons. You can see the rendered version of your branch here: https://github.com/bcgov/parks-adventure-mobile/blob/418d277169f276ad311b7a866d8183bc431db445/README.md

Reply:
Darn ☹️